test/testgpu_spinning_cube.c

```diff
@@ -261,6 +261,7 @@ CreateDepthTexture(Uint32 drawablew, Uint32 drawableh)
     depthtex_createinfo.levelCount = 1;
     depthtex_createinfo.sampleCount = render_state.sample_count;
     depthtex_createinfo.usageFlags = SDL_GPU_TEXTUREUSAGE_DEPTH_STENCIL_TARGET_BIT;
+    depthtex_createinfo.props = 0;
```

Comment:
Maybe the examples should SDL_zero() any create structs, and 0 should be a default value for all parameters?

Reply:
We're thinking about this in #163, also tracking general struct defaults at #173